README.md

```diff
@@ -1,4 +1,104 @@
+- JavaScripts
+- GitFlow
+
+## Video Presentation
+- [Click Here] (https://www.loom.com/share/c949d412bf1a49ff80c2cc45f5bee719)
+- [Click Here] (https://www.loom.com/share/e388d9622839441fb24c2aed1d7f747b)
+
+## Live Demo 
+
+- [Click Here](https://hadi111jafari.github.io/covid19-tracker/)
+
+### Github
+
+Take a look at the Covid19 Tracker page [covid19-tracker@GitHub](https://github.com/Hadi111jafari/covid19-tracker)
+
+## Development set up
+
+Clone Repository using
+
+`git clone git@github.com:Hadi111jafari/covid19-tracker.git`
+
+OR using HTTPS
+
+`git clone https://github.com/Hadi111jafari/covid19-tracker.git`
+
+move into project directory
+
+`cd covid19-tracker`
+
+Install Project dependancies using `NPM`
+
+`npm install`
+
+launch project locally with
+
+`npm start`
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
+If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+
+You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+## Author
+
+👤 **Hadi Jafari**
+
+- GitHub: [@githubhandle](https://github.com/Hadi111jafari)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/abdul-hadi-jafari)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Hadi111jafari/covid19-tracker/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse team for guidance
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please, consider fixing the link to the MIT.md file, actually, it is broken.

Reply:
Thank you for your feedback, I fixed the link